include/rapidjson/writer.h

```diff
@@ -498,6 +525,7 @@ class Writer {
     internal::Stack<StackAllocator> level_stack_;
     int maxDecimalPlaces_;
     bool hasRoot_;
+    int depthinArray_, prevDepthinArray_;
```

Comment:
These seem only be used by `PrettyWriter`. Can they be moved to `PrettyWriter` only?